compiler/noirc_evaluator/src/ssa/opt/unrolling.rs

```diff
@@ -1199,6 +1207,27 @@ mod tests {
         assert!(stats.is_small());
     }
 
+    #[test]
+    fn test_boilerplate_stats_i64_empty() {
+        // Looping 0..-1, which should be 0 iterations.
+        let ssa = brillig_unroll_test_case_6470_with_params("i64", "0", "18446744073709551615");
+        let stats = loop0_stats(&ssa);
+        assert_eq!(stats.iterations, 0);
+        assert_eq!(stats.unrolled_instructions(), 0);
+    }
+
+    #[test]
+    fn test_boilerplate_stats_i64_non_empty() {
+        // Looping -4..-1, which should be 3 iterations.
+        let ssa = brillig_unroll_test_case_6470_with_params(
+            "i64",
+            "18446744073709551612",
```

Comment:
I see it is `-4` from the comment above, but perhaps we could do this through code? Or at least just an additional comment that this is how Noir represents negative numbers.

Reply:
Changed it to `u64::MAX - 3` and also added a comment.